### ISSUE TITLE
support for linking with different games/saves

### ISSUE THREAD

**Issue by Royerson** [libretro/tgbdual-libretro#4]
Gotta trade them pokemans

Being able to load a different rom for each player would be useful for pokemon and maybe other cases, and seperate saves even when playing the same rom would also obviously be useful.

**Comment by JamesTheAwesomeDude** [comment]
Seconding this. The standalone TGB Dual supports different-ROM linking; it would be an essential feature for this core.

According to http://www.pokemonemulator.com/gameboy-color.htm,

>TGB Dual 7 ENG can also be used to play and link GBC Zelda Oracle of Seasons / Oracle of Ages games

**Comment by lifning** [comment]
https://github.com/libretro/tgbdual-libretro/blob/master/libretro/libretro.cpp#L395
is the function that does this.
Does RetroArch still expose a way to use retro_load_game_special()?

**Comment by fr500** [comment]
yes it supports it but it has changed a lot, there is now a subsystem api, this core hasn't been updated to support that api

**Comment by JamesTheAwesomeDude** [comment]
@fr500 So this is possible, just not implemented yet?

**Comment by fr500** [comment]
yes, the only core that implements this API at is bsnes for sgb/bsx

**Comment by fr500** [comment]
also RA doesn't really have the GUI hooked up for this yet, so only CLI support is available

**Comment by doritobob269** [comment]
So I was wondering if anything has changed with th=his feature being allowed as a "few" years have passed

**Comment by orbea** [comment]
@doritobob269 I think other cores might be the way to go now, I think Sameboy might have it, but I haven't tested it personally.

**Comment by fr500** [comment]
this one has it too.

**Comment by doritobob269** [comment]
How would one go about using multiple roms @fr500 ? Or how would one go about installing @orbea ? I am on my retropie btw.

**Comment by fr500** [comment]
Installing no idea we don't generally use retropie, just vanilla retroarch

https://www.youtube.com/watch?v=PDSXaV-YzkM&feature=youtu.be
